src/features/network/components/network-menu/network-menu.tsx

```diff
@@ -38,20 +38,11 @@ export function NetworkMenu() {
     setEditingNetwork(network)
     onOpen()
   }
-  const { activeNetwork, networks, activeId, setActiveId } = useNetworkStore(
-    s => ({
-      networks: Array.from(s.byId).sort((a, b) => {
-        const [, { name: nameA }] = a
-        const [, { name: nameB }] = b
-        const nameALower = nameA.toLowerCase()
-        const nameBLower = nameB.toLowerCase()
-        return nameALower === nameBLower ? 0 : nameALower < nameBLower ? -1 : 1
-      }),
-      activeId: s.activeId,
-      activeNetwork: s.byId.get(s.activeId),
-      setActiveId: s.setActiveId,
-    }),
-  )
+  const { activeNetwork, networks, setActiveId } = useNetworkStore(s => ({
+    networks: s.getNetworks(),
+    activeNetwork: s.getActiveNetwork(),
+    setActiveId: s.setActiveId,
+  }))
```

Comment:
All of this business logic was moved out of the view and into the state store. See below.